app.py

```diff
@@ -504,6 +510,52 @@ def init_db():
         mock_db.gen_mock_data()
     return "Database reseted"
 
+@app.route("/init", methods=['GET', 'POST'])
+def init():
```

Comment:
Dzisiejszy deploy będzie naszym ostatnim, więc wszelkie dziwne funkcje powinny zniknąć - jeżeli potrzebujesz dodać użytkowników, należy to zrobić ręcznie lub w funkcji init_db() - to wygląda na otwarty endpoint, który każdy może wywołać sobie ile razy chce